sdk/monitor/opentelemetry-exporter/src/Declarations/Contracts/Generated/Data.ts

```diff
@@ -0,0 +1,18 @@
+// THIS FILE WAS AUTOGENERATED
+import Base = require('./Base');
```

Comment:
seems like this should be using `import Base from "./base"` syntax instead of require.

Reply:
In general, even autogenerated code should use ES6 modules, I think